.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+on: [pull_request]
+
+env:
+  RUBY_VERSION: 2.7
+  RUBOCOP_VERSION: 0.90.0
+
+jobs:
+  rubocop:
+    name: runner / rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: reviewdog/action-rubocop@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-check
+          rubocop_version: ${{ env.RUBOCOP_VERSION }}
+
+  rspec:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        ports: ['5432:5432']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
```

Comment:
Can we use `.ruby-version` here?

We should be able to read it in similar to https://github.com/do-community/do-moderator-toolbox/blob/5fdc1227446477fb566d22e35d48528937a61db9/.github/workflows/gh-pages-workflow.yml#L15-L22